docs/graphql/manual/migrations/reference/metadata-format.rst

```diff
@@ -25,12 +25,239 @@ of multiple files.
+
+functions.yaml
+^^^^^^^^^^^^^^
+
+Contains the metadata related to :ref:`custom functions<custom_sql_functions>`
+
+..  code-block::  yaml
+
+    - function:
+      schema: public
+      name: search_books
+
+
+In the above example ``functions.yaml`` file consists of search_books custom sql function
+
+query_collections.yaml
+^^^^^^^^^^^^^^^^^^^^^^
+
+Conatins the information about query query collections
+
+..  code-block::  yaml
+
+  - name: allowed-queries
+    definition:
+      queries:
+      - name: test
+        query: |-
+          query test {
+            books {
+              id
+              author_id
+              title
+            }
+          }
+      - name: test2
+        query: |-
+          query test2 {
+              authors{
+                  id
+                  author_name
+              }
+          }
+    
+In above example there is only one collection ``allowed_queries`` which contains two queries ``test`` and ``test2`` .
+
+
+allow_list.yaml
+^^^^^^^^^^^^^^^
+
+Contains the metadata related to :ref:`allow lists<allow_list>`
+
+..  code-block::  yaml
+
+  - collection: allowed-queries
+
+The allowed queries are under the collection name ``allowed-queries``
+``allow_list.yaml`` contains collection name of allowed queries.
+
+
+
+actions.yaml
+^^^^^^^^^^^^
+
+Contains the metadata related to :ref:`actions<actions>`
+
+..  code-block::  yaml
+
+      actions:
+    - name: greet
+      definition:
+        kind: ""
+        handler: <base_url>/greet
+        forward_client_headers: true
+        headers:
+        - value: application/json
+          name: Content-Type
+    custom_types:
+      enums: []
+      input_objects:
+      - name: SampleInput
+      objects:
+      - name: SampleOutput
+      scalars: []
+
+
+In above example the url is specified directly to handler field
+
+..  code-block::  yaml
+
+      actions:
+    - name: greet
+      definition:
+        kind: ""
+        handler: '{{ACTION_BASE_URL}}/greet'
+        forward_client_headers: true
+        headers:
+        - value: application/json
+          name: Content-Type
+    custom_types:
+      enums: []
+      input_objects:
+      - name: SampleInput
+      objects:
+      - name: SampleOutput
+      scalars: []
+
+
+The above example is same to the previous except the url is specified directly not via environment variable.
+
+actions.graphql
+^^^^^^^^^^^^^^^
+
+Contains all the action definition and custom type definitions where the metadata information about actions will be stored in actions.yaml
+
+..  code-block:: graphql
+
+      type Query {
+        greet (
+          arg1: SampleInput!
+        ): SampleOutput
+      }
+      input SampleInput {
+        username : String!
+      }
+      type SampleOutput {
+        greetings : String!
+      }
+
+
+The above example actiions.graphql contains the definition of greet action and custom type  SampleOutput and SampleInput definitions.
```

Comment:
The above example ``actiions.graphql`` contains the definition of greet action and custom type  ``SampleOutput`` and ``SampleInput`` definitions.

Reply:
Also, is it `actiions.graphql` or `actions.graphql` ?

Reply:
It is actions.graphql